DQM/Integration/python/test/castor_dqm_sourceclient-live_cfg.py

```diff
@@ -73,72 +138,148 @@
-                           CastorBasicJetsLabel = cms.InputTag("ak7CastorJets"),
-                           CastorJetIDLabel     = cms.InputTag("ak7CastorJetID"),
-                          
-			   DataIntMonitor= cms.untracked.bool(True),
-			   TowerJetMonitor= cms.untracked.bool(False),
-
-                           DigiMonitor = cms.untracked.bool(True),
-                           DigiPerChannel = cms.untracked.bool(True), 
-                           DigiInFC = cms.untracked.bool(False),
-                          
-                           RecHitMonitor = cms.untracked.bool(True), 
-			   RecHitsPerChannel = cms.untracked.bool(True),
-
-                           ChannelQualityMonitor= cms.untracked.bool(True),
-                           nThreshold = cms.untracked.double(60),
-                           dThreshold = cms.untracked.double(1.0),
-                           OfflineMode = cms.untracked.bool(False),
-                           averageEnergyMethod = cms.untracked.bool(True),          
-
-                           PSMonitor= cms.untracked.bool(True),
-                           numberSigma = cms.untracked.double(1.5),
-                           thirdRegionThreshold =  cms.untracked.double(999999.0), # to avoid empty cells in reportSummary            
-                           EDMonitor= cms.untracked.bool(True),
-                           HIMonitor= cms.untracked.bool(True),
-                                       
-                           diagnosticPrescaleTime = cms.untracked.int32(-1),
-                           diagnosticPrescaleUpdate = cms.untracked.int32(-1),
-                           diagnosticPrescaleLS = cms.untracked.int32(-1),
-                             
-                           LEDMonitor = cms.untracked.bool(True),
-                           LEDPerChannel = cms.untracked.bool(True),
-                           FirstSignalBin = cms.untracked.int32(0),
-                           LastSignalBin = cms.untracked.int32(9),
-                           LED_ADC_Thresh = cms.untracked.double(-1000.0)      
-                           )
+       ### GLOBAL VARIABLES
+       debug = cms.untracked.int32(0), # make debug an int so that different
+       # values can trigger different levels of messaging
+       # 0 - no debug infomration
+       # 1 - Program flow in/out
+       # 2 - All major computations
+       # Turn on/off timing diagnostic info
+       showTiming = cms.untracked.bool(False),
+       dump2database = cms.untracked.bool(False),
+       pedestalsInFC = cms.untracked.bool(False),
+       # Define Labels
+       digiLabel = cms.InputTag("castorDigis"),
+       rawLabel = cms.InputTag("rawDataCollector"), #rawDataCollector #source
+       unpackerReportLabel = cms.InputTag("castorDigis"),
+       CastorRecHitLabel = cms.InputTag("castorreco"),
+       CastorTowerLabel = cms.InputTag("CastorTowerReco"),
+       CastorBasicJetsLabel = cms.InputTag("ak7CastorJets"),
```

Comment:
I changed the label from ak7BasicJets to ak7CastorJets looking at the code here: https://github.com/cms-sw/cmssw/blob/CMSSW_7_4_X/DQM/Integration/python/test/castor_dqm_sourceclient-live_cfg.py . In fact this is one of the "minor problems" I mentioned ("in passing") in one of my e-mails: unfortunately the CastorTowerJetMonitor (the one that uses the collections we are discussing) does not show anything sensible for example in https://cmsweb.cern.ch/dqm/online-playback/ . Should one try to fix it given https://github.com/cms-sw/cmssw/pull/7499 ?